vc-got.el

```diff
@@ -745,6 +773,7 @@ It's like `vc-process-filter' but supports \\r inside S."
 ;; TODO: this could be expanded.  After a pull the worktree needs to
 ;; be updated, either with a ``got update -b branch-name'' or ``got
 ;; update -b remote/branchname'' plus a rebase.
+;; TODO: can this handle prefix argument? pull does fetch, C-u pull does update?
```

Comment:
I guess that here we should do some rebase or merging, possibly letting the user choose with an option or prefix argument, but don't use the prefix to do the update, that should be done by default I think.